core/src/com/unciv/logic/city/managers/CityConquestFunctions.kt

```diff
@@ -33,9 +34,21 @@ class CityConquestFunctions(val city: City) {
         val baseGold = 20 + 10 * city.population.population + tileBasedRandom.nextInt(40)
         val turnModifier = max(0, min(50, city.civ.gameInfo.turns - city.turnAcquired)) / 50f
         val cityModifier = if (city.containsBuildingUnique(UniqueType.DoublesGoldFromCapturingCity)) 2f else 1f
+        
+        var cityModifierNew = 1f
```

Comment:
Nah. Better make the existing `val` into `var` and multiply your new ones in, more readable.